File: front/src/app/Models/Tournament.ts
export class Tournament {
  id: number;
  name: string;
  regulations: string;
  date_start: Date;
  date_close_reg: Date;
  date_cancel_reg: Date;

  constructor(name: string, regulations: string, date_start: Date, date_close_reg: Date, date_cancel_reg: Date) {
    this.name = name;
    this.regulations = regulations;
    this.date_start = date_start;
    this.date_close_reg = date_close_reg;
    this.date_cancel_reg = date_cancel_reg;
  }
}

File: back/src/main/java/com/models/Tournament/Tournament.java
package com.models.Tournament;

import lombok.*;

import javax.persistence.*;
import java.util.Date;

@Data
@Entity
@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
public class Tournament {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "id", nullable = false, unique = true)
    private Long id;
    private String name;
    private String regulations;
    private Date date_start;
    private Date date_close_reg;
    private Date date_cancel_reg;
    private String results;
}
